plugin/src/main/java/io/fabric8/maven/plugin/BuildMojo.java

```diff
@@ -109,6 +108,12 @@
     private OpenShiftBuildStrategy buildStrategy = OpenShiftBuildStrategy.s2i;
 
     /**
+     * Should we use the project's compmile-time classpath to scan for additional enrichers/generators?
+     */
+    @Parameter(property = "fabric8.useProjectClasspath", defaultValue = "false")
+    private boolean useProjectClasspath = false;
```

Comment:
I would do it unconditionally since tbh I dont see a use case where we should not use the project classpath. In order to exclude / include enrichers one could still use a dedicated configuration for this. I'm a fan of adding config options only if absolutely needed (number of configuration options will explode anyway). Also save some documentation ;-)